MapboxMaps.podspec

```diff
@@ -22,7 +22,7 @@ Pod::Spec.new do |m|
   m.resources = 'Sources/**/*.{xcassets,strings}'
 
   m.dependency 'MapboxCoreMaps', '10.0.0-beta.19'
-  m.dependency 'MapboxCommon', '11.0.0'
+  m.dependency 'MapboxCommon', '~> 11.0'
```

Comment:
Won't the `~>` cause issues if there are mis matching versions with MapboxCoreMaps?

Reply:
For discussion with team

Reply:
Can we make a discussion ticket for this?